src/control/controlobjectscript.h

```diff
@@ -44,5 +47,6 @@ class ControlObjectScript : public ControlProxy {
     QVector<ScriptConnection> m_scriptConnections;
     const RuntimeLoggingCategory m_logger;
     CompressingProxy m_proxy;
-    bool m_skipSuperseded; // This flag is combined for all connections of this Control Object
+    bool m_skipSuperseded;
```

Comment:
Why did you delete this comment?

Reply:
i am sorry might have been deleted by mistake